src/js/components/TaskTable.js

```diff
@@ -165,6 +184,12 @@ class TaskTable extends React.Component {
     );
   }
 
+  renderHost(prop, task) {
+    let marathonTask = MarathonStore.getTaskFromTaskID(task.id);
+
+    return <TaskEndpointsList portLimit={3} task={marathonTask} />;
```

Comment:
I did add the key but I don't think that's going to help us out: https://facebook.github.io/react/docs/reconciliation.html#keys